pandas/computation/tests/test_eval.py

```diff
@@ -1411,6 +1411,22 @@ def test_multi_line_expression_not_inplace(self):
         e = a + 2""", inplace=False)
         assert_frame_equal(expected, df)
 
+    def test_multi_line_expression_local_variable(self):
+        # GH 15342
+        tm.skip_if_no_ne('numexpr')
```

Comment:
This line is a cut and paste from the test case above.  I assumed it was desired boilerplate. It is present in the three test cases above the the test case I added.  Would you like me to take some action on this?